www/js/search.js

```diff
@@ -31,15 +31,15 @@ const searchInputs = h('div#search-container', [
     onfocus: e => module.exports.focusSearch(e),
     onkeyup: e => module.exports.typeSearch(e),
   }),
-  h('div#search-bg'),
-  h(
-    'button#search-options-toggle',
-    {
-      type: 'button',
-      onclick: e => module.exports.toggleSearchOptions(e),
-    },
-    h('i.fa.fa-cog'),
-  ),
+  h('span', 'Ang'),
+  h('input#ang-input.gurmukhi', {
```

Comment:
I think we should also consider clearing this when other searches are done. it could get confusing if the ang stil says some number but the user search for letters anywhere. can be done later.